homework_1/task3/main.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+)
+
+const pokemonURL = "https://pokeapi.co/api/v2/pokemon/"
```

Comment:
The whole pokemon-related stuff should be moved to a different file/files from `main.go`.

Reply:
I'm not really sure what do you mean by moving it to a different file? Should the link be stored in a file? Or should the output be stored in a `.txt` file?

Reply:
Sorry, I wasn't clear enough. I meant everything except `main()` function. Because currently main.go has 200 lines with lots of logic inside, it doesn't make sense. So you can move everything except `main()` to a different package (e.g. `pokemon`) and create a file within that package.